vasyl_varyshchuk/Week_2/task_3/show_pids.py

```diff
@@ -0,0 +1,16 @@
+"""
+This module calls a bash script that searches for running
+python processes and displays PID's of these processes
+"""
+
+import argparse
+from subprocess import call
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--search', help='search for python process')
+args = parser.parse_args()
+
+argument = str(args.search)
```

Comment:
What is `args.search`?

Reply:
In my case, args.search returns command line argument for flag --search.